### ISSUE TITLE
Popsplit bug. (Like the millionth one lol)

### ISSUE THREAD

**Issue by AlexHGaming** [Megabyte918/MultiOgar-Edited#413]
This may make no sense to you, but I'm going to try explain the best I can:
Basically
When doing popsplits, they work and.. well yeah
But there is this one bug, that happens **Sometimes** not the size of the cells, not positioning.
(here comes the hard explanation)
SOMETIMES when the 4th part enters enemy, the 3rd part approaches and eats around 14 parts.
But for some reason, when the 3rd part approaches to eat the bigger part, for some reason, that bigger part kills me when the 3rd cell is bigger. I'm sorry for a bit of bad explanation
I can't record because it drowns my CPU usage and lags to the high heavens.
But some people might face this...
Basically what I'm trying to say is, 3rd part is bigger then the enemy's last big part, but enemy manages to eat the 4th part.
This doesn't happen occasionally but at rare times this happens.
If this makes no sense to you (which it make actually make sensee) I'll translate it the best I can.
How do you fix this bug?

**Comment by Megabyte918** [comment]
I am aware of this and I've been working on the fix for a while. Please do not make any more issues about popsplits.
